test/test_KubeLibrary.py

```diff
@@ -319,9 +326,9 @@ def test_get_role_bindings_in_namespace(self, mock_lnp):
         role_bindings = kl.get_role_bindings_in_namespace('default')
         self.assertEqual(['read-pods'], [item for item in role_bindings])
 
-    @mock.patch('kubernetes.client.CoreV1Api.list_namespaced_service')
-    def test_get_services_in_namespace(self, mock_lnp):
-        mock_lnp.side_effect = mock_list_namespaced_services
+    @mock.patch('kubernetes.client.CoreV1Api.list_namespaced_config_map')
```

Comment:
I think you are replacing here the test for `list_namespaced_service` instead of adding an additional test for `list_namespaced_config_map`

Reply:
This issue is resolved , can you please review.